### ISSUE TITLE
issue with how .inp is written when TYPE = IMPUTATION

### ISSUE THREAD

**Issue by wepelham3** [michaelhallquist/MplusAutomation#88]
Bug seems to have been introduced between v0.7 and v0.7-3

When `imputed = TRUE`, the `.dat` file that contains a list of the individual imputed datasets is the `modelout` parameter with `_implist` appended.

But the `.inp` is written such that `FILE` equals the modelout parameter, ***without*** `_implist` appended.

So Mplus cannot find the datafiles.

````
library(mice)
library(MplusAutomation)

imp <- mice(nhanes)

imp <- complete(imp, action = "long")

implist <- split(imp, imp$.imp)

mod <- mplusObject(rdata = implist,
                   imputed = TRUE,
                   MODEL = "bmi on age;")

mplusModeler(mod,
             modelout = "model.inp",
             writeData = "always", run = 1)
````

Should produce an error in Mplus file like this:

````
*** ERROR in DATA command
  The file specified for the FILE option cannot be found.  Check that this
  file exists:  model.dat
````

I am using Mplus v7.4 if relevant.

**Comment by JWiley** [comment]
@wepelham3 Thanks will check into this and try to get a fix up soon.

**Comment by JWiley** [comment]
@wepelham3 I believe this is fixed now in commit 9209ea537369eb9e459323830ebad36d45669cf1
